packages/lite-insight/src/attribution/multiDimension.ts

```diff
@@ -0,0 +1,136 @@
+import { Datum } from '../interface';
+
+type disassembleInfoSub = {
+  startValue: number;
+  endValue: number;
+  diff: number;
+  contribution: number;
+};
+
+type disassembleInfo = {
+  total: number;
+  detail: Object;
```

Comment:
Type declaration for "detail" could be more specific?

Reply:
declared.